Makefile

```diff
@@ -195,6 +195,16 @@ semgrep:
 	@semgrep --exclude "securedrop/tests/" --error --strict --metrics off --max-chars-per-line 200 --verbose --config "p/r2c-security-audit" securedrop
 	@echo
 
+
+# check dependencies in Cargo.lock
+.PHONY: rust-audit
+rust-audit:
+	@echo "███ Running Rust dependency checks..."
+	@cargo install cargo-audit
```

Comment:
I assume it is intentional that you didn't pin the version here, which I agree with.

Reply:
*pulls collar nervously*